searx/searx
# Fix selects in preferences so they look consistent

## What does this PR do?

1. Make all select inputs use the same style.
2. Fix triangle in rtl locales.

## Why is this change important?

1. In the preferences page the language selector currently looks different from the rest of the selectors in the same page.
2. When using a rtl locale, the triangle is placed directly over the text which makes it harder to read.

## How to test this PR locally?

1. Go to the preferences page and verify that all selects look the same.
2. Change the interface language to either Arabic, Hebrew or Persian/Farsi and check that all selects look the same, with the triangles at the left end.

## Author's checklist

Apparently, the pointhi style in oscar doesn't have the `custom-select` class at all.
